rules/windows/builtin/application/screenconnect/win_app_remote_command_execution.yaml

```diff
@@ -0,0 +1,24 @@
+title: Remote Access Tool - ScreenConnect Remote Tool Execution
+id: 076ebe48-cc05-4d8f-9d41-89245cd93a14
+related:
+    - id: b1f73849-6329-4069-bc8f-78a604bb8b23
+      type: similar
+status: experimental
+description: ScreenConnect RMM has feature to remotely execute binaries on a target machine.
+author: Ali Alwashali
+date: 2023/10/10
+tags:
+    - attack.execution
+    - attack.t1059.003
+logsource:
+    service: application
+    product: windows
+detection:
+    selection:
+        Provider_Name: 'ScreenConnect'
+        EventID: 200
+        Data|contains: 'Executed command of length'
```

Comment:
Are there any other detailed provided in this log? can you give an example.
Maybe we can create a variation of this rule with other conditions

Reply:
![image](https://github.com/SigmaHQ/sigma/assets/22593441/1809546d-d817-406c-b9e8-5039ecbc0839)